nx/include/switch/services/nifm.h

```diff
@@ -16,6 +16,26 @@ typedef enum {
     NifmServiceType_Admin          = 3, ///< Initializes nifm:a.
 } NifmServiceType;
 
+typedef enum {
+    NifmInternetConnectionType_WiFi     = 1, ///< Wi-Fi connection is used.
+    NifmInternetConnectionType_Ethernet = 2, ///< Ethernet connection is used.
+} NifmInternetConnectionType;
+
+typedef enum {
```

Comment:
Does this really need to be an enum?

Reply:
What else would it be?

Reply:
Plain u32?

Reply:
Well I don't mind, it's just that I don't see any reason not to make it an enum, the meaning is clear that way

Reply:
Meant NifmWifiStrengthBars not NifmInternetConnectionType.

Reply:
Oh that makes sense. GitHub didn't make it clear where the review actually was